CHANGELOG.latest.md

```diff
@@ -1,49 +1,20 @@
-- Ninth beta for RevenueCat framework 🎉
+- Tenth beta for RevenueCat framework 🎉
```

Comment:
I'd recommend side-by-side for this file, or otherwise just viewing the final rendered version, which is in the PR description